go.mod

```diff
@@ -25,18 +25,16 @@ require (
 	istio.io/client-go v1.18.1 //There was no version update on 1.18.2
 	istio.io/istio v0.0.0-20230721203100-0183f2886bc0
 	istio.io/pkg v0.0.0-20230524020242-1015535057be
-	k8s.io/api v0.27.4
-	k8s.io/apiextensions-apiserver v0.27.4
-	k8s.io/apimachinery v0.27.4
-	k8s.io/client-go v0.27.4
-	k8s.io/kubectl v0.27.4
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
+	k8s.io/api v0.28.0
+	k8s.io/apiextensions-apiserver v0.28.0-alpha.4
```

Comment:
The version is forced by Istio dependency `github.com/prometheus/common@v0.42.0`